kafka-streams-quickstart/README.md

```diff
@@ -32,14 +32,20 @@ A Docker Compose file is provided for running all the components.
 Start all containers by running
 
 ```bash
-docker-compose up --build
+docker-compose up -d --build
+```
+
+Due to an [an issue](https://github.com/smallrye/smallrye-reactive-messaging/issues/128) in the SmallRye Reactive Messaging component it might be that the producer application isn’t sending events if it was started before Apache Kafka was up. In this case, simply restart the producer (run from another shell window, keeping the other services running):
+
+```bash
+docker-compose stop producer && docker-compose start producer
 ```
 
 Now run an instance of the _debezium/tooling_ image which comes with several useful tools such as _kafkacat_ and _httpie_:
 
 ```bash
 docker run --tty --rm -i \
-        --network kafka-streams-quickstart_default \
+        --network kafkastreamsquickstart_default \
```

Comment:
That's interesting, I could swear I've ran this many times. Has the default network name format changed perhaps? Eventually it would be better to use a fixed (and shorter) name in the Compose file, avoiding any confusion.

Reply:
The [kafak-streams guide](https://quarkus.io/guides/kafka-streams-guide) is using `docker run --tty --rm -i --network ks debezium/tooling:1.0`

So maybe we can armonize both by using the same one in this quistart repo ?

Reply:
fixed

Reply:
Ok, now I updated my docker-compose version from 1.16.1 to 1.24.1, the network has the right name (with dashes).
So fixing the network name is really a good idea to be consistent.